Makefile

```diff
@@ -4,6 +4,10 @@ build:
 	@echo "Building docker container..."
 	@./build-gauntlt.sh
 
+build-dev:
+	@echo "Building docker development container..."
```

Comment:
why do we need a docker-dev file?

Reply:
It's to build gauntlt from a different branch or repo, for example while developping a new feature.